requirements.txt

```diff
@@ -1,5 +1,7 @@
-cript==0.4.7
+cript==0.5.0
 openpyxl==3.0.10
 pandas==1.4.4
-PyYAML==6.0
-tqdm==4.64.0
\ No newline at end of file
+black==22.3.0
+pre-commit==2.17.0
+eel==0.14.0
+pyinstaller==5.4.1
```

Comment:
Separating which dependencies are required for the code to function from others is quite useful. I don't see a reason to deliberately move away from this, so let's recreate the `requirements_dev.txt` file.